### PR TITLE
Feat/registration frontend

### PR DESCRIPTION
**Pull request by xReddskie**
OVERVIEW
Created Front End for registration page

NOTES
npm install -D tailwindcss postcss autoprefixer sass flowbite
npx tailwindcss init -p

SNAPSHOTS

image

### CODE REVIEW

File: src/resources/css/app.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```
Comment (hipe01mark) [inline src/resources/css/app.css:3]: trail spacing!

File: src/resources/scss/app.scss [hunk 2]
```diff
@@ -0,0 +1 @@
+@use './auth/register';
```
Comment (hipe01mark) [inline src/resources/scss/app.scss:1]: trail spacing!

File: src/tailwind.config.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      },
+      boxShadow: {
+        customshadow: '0px 0px 26px 0px rgba(0, 0, 0, 0.8)',
+      },
+    },
+  },
+  plugins: [
+    require('flowbite/plugin')
+  ],
+}
```
Comment (hipe01mark) [inline src/tailwind.config.js:27]: trail spacing!

Prevent this.
image

File: src/vite.config.js [hunk 4]
```diff
@@ -4,8 +4,8 @@ import laravel from 'laravel-vite-plugin';
 export default defineConfig({
     plugins: [
         laravel({
-            input: ['resources/css/app.css', 'resources/js/app.js'],
+            input: ['resources/css/app.css', 'resources/js/app.js', 'resources/scss/app.scss'],
             refresh: true,
         }),
     ],
-});
+});
```
Comment (hipe01mark) [inline src/vite.config.js:11]: trail spacing!

Prevent this.
image

**Comment by xReddskie** [comment]
Closing this PR, wrong branch pushed. I apologize.

**Pull request closed without merging**